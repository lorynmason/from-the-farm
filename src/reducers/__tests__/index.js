import { rootReducer } from '../index';

describe('rootReducer', () => {
  it('should match snapshot', () => {
    expect(rootReducer).toMatchSnapshot();
  });
});