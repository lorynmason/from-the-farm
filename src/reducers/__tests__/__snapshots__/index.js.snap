// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rootReducer should match snapshot 1`] = `[Function]`;
